server/eclipse-project/src/main/java/us/freeandfair/corla/auth/Authentication.java

```diff
@@ -68,22 +64,8 @@ private Authentication() {
    */
   public static boolean isAuthenticatedAs(final Request the_request, 
                                           final AdministratorType the_type) {
-    boolean result = false;
-    final Object admin_attribute = the_request.session().attribute(ADMIN);
-    
-    if (admin_attribute instanceof Administrator) {
-      final Administrator admin = (Administrator) admin_attribute;
-      result = admin.type() == the_type;
-      the_request.session().attribute(ADMIN, admin);
-    } else if (admin_attribute != null) {
-      // this should never happen since we control what's in the session object,
-      // but if it does, we'll clear out that attribute and thereby force another
-      // authentication
-      Main.LOGGER.error("Invalid admin type detected in session.");
-      unauthenticate(the_request);
-    }
-    
-    return result;
+    assert false;
```

Comment:
Should I interpret this to mean "this class is now deprecated"? Which is fine... but why not, then, just make the appropriate changes to the parts of the system that use it and remove it? Or, alternatively, leave it alone until things are switched over?

Reply:
(I see you did make appropriate changes to parts of the system that use it... so you might as well delete it - that'll tell you for sure if there are parts of the system you missed, for one thing :) )

Reply:
Yes, I was testing with it bottomed out.  It is now going away.